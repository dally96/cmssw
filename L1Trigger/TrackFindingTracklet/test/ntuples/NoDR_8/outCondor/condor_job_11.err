************************************************************************************************************************
						 WARNING!
 The key phase2_realistic now points to 113X_mcRun4_realistic_v7.
 Usage of this key is discretionary and users are cautioned to use it at their own risk!
 This Global Tag contains tags suited for an Inner Tracker layout with rectangular (25x100um2) pixel cells (T15,T21).
 If this is not the geometry you are expecting to use, please consult:
 https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoCondPhase2.py to retrieve the right key.
************************************************************************************************************************
16-Feb-2022 21:47:24 CET  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/6bd37924-f529-4fa1-841e-1e7ad259169c.root
%MSG-w XrdAdaptorInternal:  file_open 16-Feb-2022 21:47:25 CET pre-events
Failed to open file at URL root://eoscms.cern.ch:1094//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/6bd37924-f529-4fa1-841e-1e7ad259169c.root?xrdcl.requuid=8170bf9d-a31a-46a0-b81b-c278f1af44b8.
%MSG
%MSG-w XrdAdaptorInternal:  file_open 16-Feb-2022 21:47:25 CET pre-events
Failed to open file at URL root://eoscms.cern.ch:1094//eos/cms/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/6bd37924-f529-4fa1-841e-1e7ad259169c.root?tried=&xrdcl.requuid=a299e895-0336-4c03-8cfb-1b52e9584208.
%MSG
16-Feb-2022 21:47:25 CET  Initiating request to open file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/6bd37924-f529-4fa1-841e-1e7ad259169c.root
%MSG-w XrdAdaptor:  file_open 16-Feb-2022 21:47:27 CET pre-events
Data is served from T2_UK_SGrid_RALPP instead of original site T2_FR_GRIF_LLR
%MSG
16-Feb-2022 21:47:32 CET  Successfully opened file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/6bd37924-f529-4fa1-841e-1e7ad259169c.root
L1TrackNtupleMaker::beginJob
----- Begin Fatal Exception 16-Feb-2022 21:47:33 CET-----------------------
An exception of category 'FileFlushError' occurred while
   [0] Calling File::flush()
Exception Message:
fdatasync() failed with system error 'No such file or directory' (error code 2)
----- End Fatal Exception -------------------------------------------------
L1TrackNtupleMaker::endJob
16-Feb-2022 21:47:33 CET  Closed file root://xrootd-cms.infn.it//store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/6bd37924-f529-4fa1-841e-1e7ad259169c.root
TimeReport> Time report complete in 18.99 seconds
 Time Summary: 
 - Min event:   1e+06
 - Max event:   0
 - Avg event:   0
 - Total loop:  0
 - Total init:  18.8823
 - Total job:   18.99
 - EventSetup Lock:   0
 - EventSetup Get:   0
 Event Throughput: 0 ev/s
 CPU Summary: 
 - Total loop:  0
 - Total init:  9.26677
 - Total extra: 0
 - Total job:   9.27953
 Processing Summary: 
 - Number of Events:  0
 - Number of Global Begin Lumi Calls:  0
 - Number of Global Begin Run Calls: 0


=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 XrdAdaptor           -w file_open                              1        1
    2 XrdAdaptorInternal   -w file_open                              2        2
    3 TimeReport           -e AfterFile                              1        1
    4 Fatal Exception      -s                                        1        1
    5 fileAction           -s file_close                             1        1
    6 fileAction           -s file_open                              3        3

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 XrdAdaptor           pre-events                        
    2 XrdAdaptorInternal   pre-events       pre-events       
    3 TimeReport           EndJob                            
    4 Fatal Exception      BeforeEvents                      
    5 fileAction           EndJob                            
    6 fileAction           pre-events       pre-events       pre-events

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 3                   3
Error                   1                   1
System                  5                   5

dropped waiting message count 0
