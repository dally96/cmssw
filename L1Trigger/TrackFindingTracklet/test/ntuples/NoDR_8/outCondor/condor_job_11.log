000 (802500.011.000) 02/16 21:43:28 Job submitted from host: <188.185.121.235:9618?addrs=188.185.121.235-9618+[2001-1458-d00-49--100-36e]-9618&alias=bigbird22.cern.ch&noUDP&sock=schedd_8027_6ecd>
...
040 (802500.011.000) 02/16 21:47:10 Started transferring input files
	Transferring to host: <128.142.59.10:9618?addrs=128.142.59.10-9618+[2001-1458-301-21--100-9]-9618&alias=b7s04p7889.cern.ch&noUDP&sock=slot1_1_3635_efad_7661>
...
040 (802500.011.000) 02/16 21:47:10 Finished transferring input files
...
001 (802500.011.000) 02/16 21:47:11 Job executing on host: <128.142.59.10:9618?addrs=128.142.59.10-9618+[2001-1458-301-21--100-9]-9618&alias=b7s04p7889.cern.ch&noUDP&sock=startd_2926_9233>
...
006 (802500.011.000) 02/16 21:47:20 Image size of job updated: 286548
	99  -  MemoryUsage of job (MB)
	100908  -  ResidentSetSize of job (KB)
...
006 (802500.011.000) 02/16 21:47:33 Image size of job updated: 1968492
	99  -  MemoryUsage of job (MB)
	100916  -  ResidentSetSize of job (KB)
...
040 (802500.011.000) 02/16 21:47:33 Started transferring output files
...
040 (802500.011.000) 02/16 21:47:33 Finished transferring output files
...
005 (802500.011.000) 02/16 21:47:33 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:09, Sys 0 00:00:02  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:09, Sys 0 00:00:02  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	12468  -  Run Bytes Sent By Job
	25479  -  Run Bytes Received By Job
	12468  -  Total Bytes Sent By Job
	25479  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       51       17   1222273 
	   Memory (MB)          :       99     2000      2000 

	Job terminated of its own accord at 2022-02-16T20:47:33Z.
...
