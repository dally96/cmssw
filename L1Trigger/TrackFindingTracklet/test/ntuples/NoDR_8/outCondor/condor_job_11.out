/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test /pool/condor/dir_37466 
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=546391038
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 06:27:15
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 06:27:14
uri       : voms-cms-auth.app.cern.ch:443

subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally/CN=546391038
issuer    : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
identity  : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
type      : RFC3820 compliant impersonation proxy
strength  : 2048
path      : /afs/cern.ch/user/d/dally/private/x509up_u141138
timeleft  : 06:27:14
key usage : Digital Signature, Key Encipherment
=== VO cms extension information ===
VO        : cms
subject   : /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=dally/CN=852449/CN=Daniel Asif Ally
issuer    : /DC=ch/DC=cern/OU=computers/CN=cms-auth.web.cern.ch
attribute : /cms/Role=NULL/Capability=NULL
timeleft  : 06:27:13
uri       : voms-cms-auth.app.cern.ch:443

cmsRun /afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/customise_L1TrackNtupleMaker.py inputFiles_clear inputFiles_load=/afs/cern.ch/user/d/dally/CMSSW_12_0_0_pre4/src/L1Trigger/TrackFindingTracklet/test/ntuples/NoDR_8/pFiles_11.txt outputFile=outputL1Ntuple_11.root print
Singletons:
  filePrepend        : 
                     - String to prepend location of all files
  maxEvents          : -1
                     - Number of events to process (-1 for all)
  outputFile         : outputL1Ntuple_11.root
                     - Name of output file (if needed)
  secondaryOutputFile: 
                     - Name of second output file (if needed)
  section            : 0
                     - This section (from 1..totalSections inclusive)
  tag                : 
                     - tag to add to output filename
  totalSections      : 0
                     - Total number of sections
Lists:
  inputFiles         : ['/store/relval/CMSSW_11_3_0_pre6/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_113X_mcRun4_realistic_v6_2026D76PU200-v1/00000/6bd37924-f529-4fa1-841e-1e7ad259169c.root' ]
                     - Files to process
  secondaryInputFiles: []
                     - Second group of files to process (if needed)

using geometry D76 (tilted)
